packages/translations/src/translations/fr.json

```diff
@@ -1303,7 +1303,7 @@
     "1571575776": "Formats acceptés : pdf, jpeg, jpg et png. Taille maximale du fichier : 8 Mb",
     "1572504270": "Opération d'arrondi",
     "1572982976": "Serveur",
-    "1573429525": "Call/Put",
+    "1573429525": "D'achat/De vente",
```

Comment:
Why do we have translation changes in your PR?

Reply:
It is from the master pull

Reply:
You can revert back these 2 changes, they are unrelated to your PR and we should not push strings into the translation json files directly.